drizzlepac/processInput.py

```diff
@@ -82,7 +82,7 @@ def setCommonInput(configObj, createOutwcs=True, overwrite_dict={}):
         configObj = {'input':None,'output':None }
 
     If provided, the configObj should contain the values of all the multidrizzle parameters
-    as set by the user with TEAL. If no configObj is given, it will retrieve
```

Comment:
I would leave this as removing the mention of TEAL may make the remainder of the text not exactly make sense or the function does not work as implied.